zqe/error.go

```diff
@@ -21,6 +21,7 @@ const (
 	Exists
 	Invalid
 	NotFound
+	NoCredentials
```

Comment:
Doesn't look like this is being used anywhere, consider dropping it?

Reply:
Lies. It's used in ppl/zqd/handlers.go.